test/generateOptions.test.js

```diff
@@ -0,0 +1,37 @@
+const rewire = require('rewire');
```

Comment:
We should not use `rewire`. The author(s) of the package have said that their module is not compatible with faking modules that make use of `const` (https://github.com/jhnns/rewire):

>Please note: The current version of rewire is not compatible with const or babel. See Limitations.

Reply:
If you'd like to use a mock library, my personal recommendation would be [Sinon](http://sinonjs.org/).